extensions/amp-list/amp-list.md

```diff
@@ -282,33 +282,62 @@ We recommend using `binding="no"` or `binding="refresh"` for faster performance.
 If `binding` attribute is not provided, default is `always`.
 
 ## Experimental: Infinite Scroll (amp-list-load-more)
```

Comment:
Experimental: Load More and Infinite Scroll ...

Reply:
Done.